wca-regulations.md

```diff
@@ -496,7 +495,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
 
 - F1) Standard speed solving procedures are followed, as described in [Article A](regulations:article:A) (Speed Solving). Additional regulations that supersede the corresponding procedures in [Article A](regulations:article:A) are described below.
 - F2) The judge places the scrambled puzzle onto the mat in a standing position.
-- F3) At the end of the inspection period, the competitor places the puzzle onto the mat in a standing position. The competitor must not change the positions of any pins from their scrambled positions before the beginning of the solve. Penalty: disqualification of the attempt (DNF).
+- F3) At the end of the inspection period, the competitor places the puzzle in a standing position. The competitor must not change the positions of any pins from their scrambled positions before the beginning of the solve. Penalty for putting the puzzle not in a standing position, or for changing the positions of any pins: disqualification of the attempt (DNF).
```

Comment:
Sorry for being picky about this, but shouldn't we write "not doing something this way" instead of "doing something not this way" ?
I think "Penalty for not putting the puzzle in a standing position" would sound better, but I'm not sure.